frontend/src/containers/Layout.tsx

```diff
@@ -18,46 +18,42 @@ interface DispatchToProps {
 
 type Props = OwnProps & DispatchToProps;
 
-class LayoutWrapper extends React.Component<Props> {
-
-  render() {
+const LayoutWrapper = (props: Props) => {
     return (
-      <Layout className="layout">
-        <Header>
-          <Menu theme="dark" mode="horizontal"
-                defaultSelectedKeys={['2']}
-                style={{ lineHeight: '64px' }}>
-            {
-              this.props.isAuthenticated ?
-              <Menu.Item key="2" onClick={this.props.logout}>
-                Logout
-              </Menu.Item>
-              :
-              <Menu.Item key="2">
-                <Link to="/login">Login</Link>
-              </Menu.Item>
-            }
-            <Menu.Item key="1">
-              <Link to="/">Home Page</Link>
-            </Menu.Item>
-          </Menu>
-        </Header>
-        <Content style={{ padding: '50px', width: '80%', margin: 'auto'}}>
-          <div>{this.props.children}</div>
-        </Content>
-        <Footer>
-        </Footer>
-      </Layout>
+        <Layout className="layout">
+            <Header>
+                <Menu theme="dark" mode="horizontal"
+                      defaultSelectedKeys={['1']}
+                      style={{ lineHeight: '64px' }}>
+                    <Menu.Item key="1">
+                        <Link to="/"><span style={{ fontSize: "1.2em", fontWeight: "bold"}}>BlogChain</span></Link>
```

Comment:
pick some styling rule for jsx, and keep consistent. Imo it would look better like
```
<Menu
  arg1='smth'
  arg2='stmh'
>
```
(new line after every argument if you have more than 1-2 e.g.)

Also - inline styles are not good approach, so later on think about putting styles into separate files.
Also - file path indicates it's a container, but there is nothing logical about it, so - I would move it to componentes instead of containers